packages/paste-core/components/badge/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@twilio-paste/badge",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "alpha",
```

Comment:
Is this truly alpha? or production?

Reply:
I used a script in `package.json` to generate the component package directory and this is one of the defaults. I know we are filing this under `stage 1` for now, so this should probably be production when it is ready to be released?

Reply:
yeah, I think the package-maker script pre-dates our term changes. I think this should be production.

Reply:
mmhmmm production pls